src/jdk.jpackage/share/classes/jdk/jpackage/internal/resources/HelpResources.properties

```diff
@@ -241,8 +241,16 @@ MSG_Help_mac_launcher=\
 \          (absolute path or relative to the current directory).\n\
 \          If not specified, the standard keychains are used.\n\
 \  --mac-signing-key-user-name <team name>\n\
-\          Team name portion in Apple signing identities' names.\n\
-\          For example "Developer ID Application: "\n\
+\          Team or user name portion of Apple signing identities.\n\
+\  --mac-app-store\n\
+\          Indicates that the jpackage output is intended for the\n\
+\          Mac App Store.\n\
```

Comment:
Maybe `macOS App Store`?

Reply:
Although I do see it referenced both ways (even by Apple) I see 10 times as many entries in Google for "Mac App Store" (1.7 billion) vs MacOS App Store (155 million)